source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/ray_caster/patterns/patterns_cfg.py

```diff
@@ -96,6 +96,53 @@ class PinholeCameraPatternCfg(PatternBaseCfg):
     height: int = MISSING
     """Height of the image (in pixels)."""
 
+    @classmethod
+    def from_intrinsic_matrix(
+        cls,
+        focal_length: float,
+        intrinsic_matrix: list[float],
+        width: int,
+        height: int,
+    ) -> PinholeCameraPatternCfg:
+        """Create a PinholeCameraPatternCfg from an intrinsic matrix. The intrinsic matrix is a 3x3 matrix that
+        defines the mapping between the 3D world coordinates and the 2D image. The matrix is defined as:
+
+        .. math::
+            \\begin{bmatrix}
+            f_x & 0 & c_x \\\\
+            0 & f_y & c_y \\\\
+            0 & 0 & 1
+            \\end{bmatrix}
+
```

Comment:
If you use math symbols, you can use "r" to tell docstrings about that. Then you don't need to do double `\\` anymore. Also the first summary line should be a single sentence. Anything more goes into a paragraph.

```suggestion
        r"""Create an :class:`PinholeCameraPatternCfg` instance from an intrinsic matrix. 
        
        The intrinsic matrix is a 3x3 matrix that defines the mapping between the 3D world coordinates and
        the 2D image. The matrix is defined as:

        .. math::
            \begin{bmatrix}
            f_x & 0 & c_x \\
            0 & f_y & c_y \\
            0 & 0 & 1
            \end{bmatrix}
            
        where :math:`f_x` and :math:`f_y` are the focal length along x and y direction, while :math:`c_x` and :math:`c_y` are the
        principle point offsets along x and y direction respectively.
```